src/assets/stack-blitz/src/index.html

```diff
@@ -1,13 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block" rel="stylesheet">
-  <title>{{title}}</title>
-</head>
-<body class="mat-app-background">
```

Comment:
Did the default background color change now?

Reply:
probably will be different in dark mode. I'm re-adding it